syntax = "proto3";

package flashsale;

message FlashsaleRequest {
  string usrname = 1;
  string usrkey = 2;
}

message FlashsaleResponse {
  string result = 1;
}

service FlashsaleService {
  rpc flashsale(FlashsaleRequest) returns (FlashsaleResponse);
}
